dvc/repo/experiments/utils.py

```diff
@@ -115,3 +115,27 @@ def fix_exp_head(scm: "Git", ref: Optional[str]) -> Optional[str]:
         if name == "HEAD" and scm.get_ref(EXEC_BASELINE):
             return "".join((EXEC_BASELINE, tail))
     return ref
+
+
+def get_exp_ref_list(
+    repo, exp_name: str, git_remote: Optional[str] = None
+) -> List[ExpRefInfo]:
```

Comment:
I don't think there will be any use case where we need the list of multiple possible experiments returned by this function. In the event that we wanted multiple matches, we would just use `exp_refs_by_name` directly. If you want to reorganize it this way, this function should probably return a single experiment, and it should raise an appropriate "ambiguous name" error if there is more than one possible matching experiment.

Also, all of the functions in this `utils` module are just doing Git operations, so they take SCM/Git instances as a parameter, rather than a DVC `Repo` instance. It should be the same for this function as well.

To keep parity with the other git libraries we use, it may also make more sense to name it something like `resolve_exp_ref`.

Reply:
Are there any other files for the common function of exp?